### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by ajhyndman** [JenShin-368/GenshinImpactCalculator#75] [Ideas]
Hi!  Thanks for building this tool.  It has already been really useful for me.

I am also a professional JavaScript developer.  I want to volunteer to help out, esp. with new character releases and updates.  Are you interested in any open source contributions to the project?

**Comment by JenShin-368** [comment]
Hi, yes. I plan to move some of the code to GitHub eventually (but will probably have to wait a couple of months).

**Reply by ajhyndman** [reply]
Cool!  I'll keep an eye out.
